backend/src/modules/boards/services/update.board.service.ts

```diff
@@ -483,7 +494,7 @@ export default class UpdateBoardServiceImpl implements UpdateBoardServiceInterfa
 	async updatePhase(boardPhaseDto: BoardPhaseDto) {
 		try {
 			const { boardId, phase } = boardPhaseDto;
-			await this.boardModel
+			const board = await this.boardModel
 				.findOneAndUpdate(
 					{
 						_id: boardId
```

Comment:
```suggestion
			const {
				slackEnable,
				phase: currentPhase,
				team
			} = await this.boardRepository.updatePhase(boardId, phase)
```
In the `board repository` create a new method that knows how to update the board phase.

```
updatePhase(boardId: string, phase: BoardPhases): Promise<Board> {
		return this.findOneByFieldAndUpdate(
			{
				_id: boardId
			},
			{
				phase
			},
			{ new: true },
			{
				path: 'team',
				select: 'name users -_id'
			}
		);
	}
```

I didn't catch this in the previous pull request but instead of using the `BoardModel`, please use the board repository. The repository should have the responsibility to know how to update the board phase